docker-compose.yml

```diff
@@ -7,9 +7,11 @@ services:
         - 4040:4040
```

Comment:
Many users do not run the github app. Can you test this without a github app and ensure it works as expected? Otherwise if an issue comes up, we'd have to revert

Reply:
Great thank you!

Yes hopefully no other impact.